getArtifact.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+here=$(realpath "$0")
+here=$(dirname "$here")
+cd "$here"
+rm -rf artifacts
+mkdir artifacts
+echo "Downloading Arkcase core artifacts $ARKCASE_VERSION"
+aws s3 cp "s3://arkcase-container-artifacts/arkcase-core/${ARKCASE_VERSION}/artifacts/" artifacts/ --recursive
```

Comment:
What artifacts are we downloading here? Is it only ArkCase artifacts or also the dependencies artifacts (Alfresco, Solr, ActiveMQ,...)?  
All artifacts could be downloaded from their release site. For ArkCase it's GitHub. The other dependencies have their own sites.

Reply:
only Arkcase application war file and mariadb sql driver.